pkg/oc/cli/cmd/newapp.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"bytes"
+	coreerror "errors"
```

Comment:
fyi: very rare in our codebase to rename core go imports, best avoided.  We'll get rid of this import below